### ISSUE TITLE
Compatibility with mini Linux pc

### ISSUE THREAD

**Issue by simonante** [matteodelabre/rmvncclient#39]
This may work fine with a mini linux pc as a main screen.
What do you think? I thought I may try this.

**Comment by matteodelabre** [comment]
Hi @simonante, could you elaborate on what specific setup you have in mind? In principle any Linux system should work since you just need to run a VNC server.

**Comment by simonante** [comment]
Any setup, as atm yet deciding to buy one. 
If I have the option to choose, linux is probably most optimal for vnsee, right? 
I am macos user. Seems not functional with macos.

**Comment by matteodelabre** [comment]
Yes, I'd recommend something that can run Linux. It may be possible to setup macOS for use with VNSee but I cannot help with that unfortunately as I don't have experience with that system.

Just to clarify, the general idea here is to run a headless system (ie without a screen attached) and connect to it with VNSee?

**Comment by simonante** [comment]
Exactly, no screen attached and connect with vnsee.
